git_machete/cli.py

```diff
@@ -222,8 +222,11 @@ def create_cli_parser() -> argparse.ArgumentParser:
         parents=[common_args_parser])
     github_parser.add_argument(
         'subcommand', choices=['anno-prs', 'checkout-prs', 'create-pr', 'retarget-pr'])
-    github_parser.add_argument('pr_no', nargs='?')
```

Comment:
Whoops also I think `checkout-prs` with no arg should not be allowed right?
```
$ git machete github checkout-prs 
Fetching origin...
Traceback (most recent call last):
  File "/usr/local/bin/git-machete", line 4, in <module>
    __import__('pkg_resources').run_script('git-machete==3.5.1', 'git-machete')
  File "/usr/lib/python3.10/site-packages/pkg_resources/__init__.py", line 651, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python3.10/site-packages/pkg_resources/__init__.py", line 1455, in run_script
    exec(script_code, namespace, namespace)
  File "/usr/local/lib/python3.10/site-packages/git_machete-3.5.1-py3.10.egg/EGG-INFO/scripts/git-machete", line 19, in <module>
  File "/usr/local/lib/python3.10/site-packages/git_machete-3.5.1-py3.10.egg/git_machete/cli.py", line 816, in main
  File "/usr/local/lib/python3.10/site-packages/git_machete-3.5.1-py3.10.egg/git_machete/cli.py", line 632, in launch
  File "/usr/local/lib/python3.10/site-packages/git_machete-3.5.1-py3.10.egg/git_machete/client.py", line 1927, in checkout_github_prs
UnboundLocalError: local variable 'pr' referenced before assignment
```

Reply:
This was because default `[]` in `pr_no` argument. fixed.